src/connectors/meraki.py

```diff
@@ -0,0 +1,186 @@
+        'title': "Meraki Network Ids Whitelist",
+        'prompt': "Whitelist of Network Ids",
+        'type': 'list',
+        'secret': True,
+        'required': True,
+    },
+]
+
+LANDING_TABLE_COLUMNS_CLIENT = [
+    ('INSERT_ID', 'NUMBER IDENTITY START 1 INCREMENT 1'),
+    ('SNAPSHOT_AT', 'TIMESTAMP_LTZ(9)'),
+    ('RAW', 'VARIANT'),
+    ('ID', 'VARCHAR(256)'),
+    ('MAC', 'VARCHAR(256)'),
+    ('DESCRIPTION', 'VARCHAR(256)'),
+    ('MDNS_NAME', 'VARCHAR(256)'),
+    ('DHCP_HOSTNAME', 'VARCHAR(256)'),
+    ('IP', 'VARCHAR(256)'),
+    ('SWITCHPORT', 'VARCHAR(256)'),
+    ('VLAN', 'INT'), 
+    ('USAGE_SENT', 'INT'),
+    ('USAGE_RECV', 'INT'),
+    ('SERIAL', 'VARCHAR(256)'),
+    ('NETWORK_RAW', 'VARIANT')
+]
+
+# LANDING_TABLE_COLUMNS_DEVICE = [
+#     ('INSERT_ID', 'NUMBER IDENTITY START 1 INCREMENT 1'),
+#     ('SNAPSHOT_AT', 'TIMESTAMP_LTZ(9)'),
+#     ('RAW', 'VARIANT'),
+#     ('SERIAL', 'VARCHAR(256)'),
+#     ('ADDRESS', 'VARCHAR(256)'),
+#     ('NAME', 'VARCHAR(256)'),
+#     ('NETWORK_ID', 'VARCHAR(256)'),
+#     ('MODEL', 'VARCHAR(256)'),
+#     ('MAC', 'VARCHAR(256)'),
+#     ('LAN_IP', 'VARCHAR(256)'),
+#     ('WAN_1_IP', 'VARCHAR(256)'),
+#     ('WAN_2_IP', 'VARCHAR(256)'),
+#     ('TAGS', 'VARCHAR(256)'),
+#     ('LNG', 'FLOAT'),
+#     ('LAT', 'FLOAT'),
+# ]
+
+
+def get_data(url: str, token: str, params: dict = {}) -> dict:
+    headers: dict = {
+        "Content-Type": "application/json",
+        "Accept": "application/json",
+        "X-Cisco-Meraki-API-Key": f"{token}",
+    }
+    try:
+        log.debug(f"Preparing GET: url={url} with params={params}")
+        req = requests.get(url, params=params, headers=headers)
+        req.raise_for_status()
+    except HTTPError as http_err:
+        log.error(f"Error GET: url={url}")
+        log.error(f"HTTP error occurred: {http_err}")
+        raise http_err
+    log.debug(req.status_code)
+    return req.json()
+
+
+def connect(connection_name, options):
+    landing_table_client = f'data.meraki_{connection_name}_connection_client'
+    # landing_table_device = f'data.meraki_{connection_name}_connection_device'
+    options['network_id_whitelist'] = options.get('network_id_whitelist', '').split(',')
+
+    # comment = yaml_dump(module='meraki', **options)
+
+    db.create_table(name=landing_table_client,
+                    cols=LANDING_TABLE_COLUMNS_CLIENT, comment="comment") # TODO: Change back to comment=comment
+    db.execute(f'GRANT INSERT, SELECT ON {landing_table_client} TO ROLE {SA_ROLE}')
+    db.create_table(name=landing_table_device,
+                    cols=LANDING_TABLE_COLUMNS_DEVICE, comment=comment)
+    db.execute(f'GRANT INSERT, SELECT ON {landing_table_device} TO ROLE {SA_ROLE}')
+    return {
+        'newStage': 'finalized',
+        'newMessage': "Meraki ingestion tables created!",
+    }
+
+
+def parse_number(value):
+    if value:
+        return value
+    return None
+
+
+def ingest(table_name_client, landing_table_device, options):
```

Comment:
```suggestion
def ingest(table_name, options):
```
I think the way this works means this has to be a function that looks like the suggestion